src/BUILD

```diff
@@ -1,6 +1,18 @@
 package(default_visibility=["//visibility:public"])
 load("@build_bazel_rules_typescript//:defs.bzl", "ts_library")
 
+TS24 = "//test_ts24:node_modules/typescript/lib/typescript.d.ts"
+
+# Copy the 2.4 version of typescript's typings into the project.
+# We don't want to copy a file that's licensed under an external license into
+# our source tree, and this mechanism allows the :src library to import it.
+genrule(
+    name = "typescript-2.4",
+    outs = ["typescript-2.4.d.ts"],
+    srcs = [TS24],
+    cmd = "cp $(location %s) $@" % TS24
```

Comment:
I think you can do `$(location $<)` or `$(location $(SRC))` or so, right?

Reply:
good catch